src/content/en/updates/2018/12/wakelock.md

```diff
@@ -100,78 +95,99 @@ For example, a screen wake lock implies that the app should continue running.
   </p>
 </div>
 
-
-### Get a wake lock object {: #get-wake-lock }
-
-In order to use the Wake Lock API, we need to create and initialize a
-`wakelock` object for the type of wake lock we want. Once created, the promise
-resolves with a `wakelock` object, but note, the wake lock isn’t active yet,
-it’ll need to be activated first.
-
+<div class="clearfix"></div>
+Warning: We’re currently defining the permission model for `system` wake locks.
+Until this work has finished, requests for `system` wake locks will be denied immediately.
+You can track our progress in [crbug.com/985742](https://crbug.com/985742).
+
+### Get a wake lock {: #get-wake-lock }
+
+In order to request a wake lock, you need to call the `WakeLock.request()` method
+that lives on the `window` object. You pass it the desired wake lock type as the first parameter,
+which *currently* is limited to just `'screen'`. In addition to that,
+you also need a way to abort the wake lock,
+which works through the generic `AbortController` interface.
+Therefore, you first create a new
+[`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController),
+and then pass the controller’s
+[`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
+as the second parameter to `WakeLock.request()`.
+Two things can happen next that you need to `catch`:
+
+1. The wake lock can after a while just be regularly aborted,
```

Comment:
These need to be bullets. I know you counted them. But the web.dev style guide (not to mention long-standing tech writing practice) reserves numbers for procedures.

Reply:
Makes sense.